### ISSUE TITLE
Value range after applying ToPILImage()

### ISSUE THREAD

**Issue by sahilg06** [pytorch/vision#7674]
### 📚 The doc issue

[Here](https://pytorch.org/vision/stable/generated/torchvision.transforms.v2.ToImagePIL.html#torchvision.transforms.v2.ToImagePIL:~:text=while%20preserving%20the%20value%20range.) it is written that `ToPILImage()` preserves the value range, Doesn't it multiply by 255.?

**Comment by pmeier** [comment]
You are right, the wording is confusing. What it means to convey is that we automatically adjust the value range for the user if needs be. Meaning, if you omit `mode` for a three channel image, we assume

https://github.com/pytorch/vision/blob/8324c481dd4c3096697332d76fbdc9d912f7360b/torchvision/transforms/transforms.py#L213

And since there is no floating point RGB support in PIL, we need to convert the value range to do the thing the user wants, i.e. get a PIL image that is as close as possible as the original tensor.

Note that if you have a grayscale tensor image, i.e. either of shape `(1, H, W)` or just `(H, W)`, and just explicitly set `mode="F"`, we don't perform a value range conversion 

https://github.com/pytorch/vision/blob/8324c481dd4c3096697332d76fbdc9d912f7360b/torchvision/transforms/functional.py#L290

```python
image_tensor = torch.rand(256, 256)
image_pil = ToPILImage(mode="F")(image_tensor)

torch.testing.assert_close(np.array(image_pil), image_tensor.numpy(), atol=0, rtol=0)
```

We would be happy to receive a PR clarifying that.

**Comment by pmeier** [comment]
Also, we want to have a proper listing in in the `mode` documentation

https://github.com/pytorch/vision/blob/8324c481dd4c3096697332d76fbdc9d912f7360b/torchvision/transforms/transforms.py#L211-L216

but it renders as [regular text](https://pytorch.org/vision/stable/generated/torchvision.transforms.v2.ToImagePIL.html#torchvision.transforms.v2.ToImagePIL). We probably need a blank line after the colon.

**Comment by sahilg06** [comment]
Hi @pmeier, I would be happy to create a PR for the same. 
Thank you!
